README.md

```diff
@@ -60,19 +60,13 @@ Lots more output
 ...
 ```
 
-## Initialize Git and register pre-commit hooks
+## Register pre-commit hooks
 
 The tool `pre-commit` is an industry standard that will execute a set of tests prior to completing a git commit action. For instance, pre-commit can run unit tests to verify code coverage or linters to ensure adherence to style guides. Pre-commit documentation can be found [here](https://pre-commit.com/index.html).
```

Comment:
I think it's worth adding that pre-commit needs to be installed before running the command. I installed it with pipx, so including the pipx install command might be useful.

Reply:
`pre-commit` is defined as dependency in the `project.optional_dependencies.dev` section in the pyproject.toml file. So it should be installed when the user runs `pip install .[dev]` in the previous section. 

I didn't call that out explicitly in the instructions in this section, but can certainly do so :)

Reply:
Oh that's me not following the instructions fully then, sorry. It might be worth adding a sentence about it just to make it clear to people like me.

Reply:
Makes sense to me. I tried to tidy up that section to make it more clear.